packages/flutter/lib/src/gestures/recognizer.dart

```diff
@@ -266,10 +266,11 @@ abstract class OneSequenceGestureRecognizer extends GestureRecognizer {
   @protected
   @mustCallSuper
   void resolvePointer(int pointer, GestureDisposition disposition) {
-    final GestureArenaEntry? entry = _entries[pointer];
+    final Map<int, GestureArenaEntry> localEntries = Map<int, GestureArenaEntry>.from(_entries);
```

Comment:
Why is this clone needed?

Reply:
entry. resolve() will trigger nested
calls to this function, so we should remove the entry first as following do.

Reply:
The entry in the map is a reference type here,  I think we can use it after removing it from the map without clone, I will double check:)

Reply:
I have removed the unnecessary map clone.